lib/cassava/client.rb

```diff
@@ -17,8 +17,8 @@ def insert_async(table, data)
 
     # @param [Symbol] table the table name
     # @param [Hash] A hash of column names to data, which will be inserted into the table
-    def insert(table, data)
-      statement = insert_statement(table, data)
+    def insert(table, data, options = {})
```

Comment:
```#insert_async``` should also have the same options. Also the other methods in the file use ```opts``` instead of ```options```




and have this usage:
```ruby
client.insert(table_name, data).if_not_exists
```